src/vsphere_cpi/spec/unit/cloud/vsphere/drs_rules/vm_group_spec.rb

```diff
@@ -107,4 +110,47 @@ def with_lock
       end
     end
   end
+
+  describe '#get_delete_rule_spec' do
+    before do
+      allow(datacenter_cluster).to receive(:configuration_ex).and_return(configuration_ex)
+    end
+    context 'when no rule exists' do
+      before do
+        allow(configuration_ex).to receive(:rule).and_return(nil)
```

Comment:
I think this returns an empty array, should confirm the with an actual call and setup the test accordingly

Reply:
added one more test to check empty. letting nil stay.